### PR TITLE
Fixes issue #64, add CORS middleware to project

### PR DESCRIPTION
**Pull request by jaymcgrath**
This should resolve any CORS issues. can @DingoEatingFuzz confirm?

### CODE REVIEW

**Comment by DingoEatingFuzz** [comment]
I'm gonna check it out right now! Will report back.

File: backend/backend/settings.py [hunk 1]
```diff
@@ -1,137 +1,146 @@
...
+    'corsheaders.middleware.CorsMiddleware',
+    'django.middleware.common.CommonMiddleware',
+    'django.middleware.csrf.CsrfViewMiddleware',
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.messages.middleware.MessageMiddleware',
+    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+]
+
+# Turn on CORS headers for API urls only
+CORS_URLS_REGEX = r'^housing_api/.*$'
```
Comment (DingoEatingFuzz) [inline backend/backend/settings.py]: This needs a slash before `housing_api`.

```py
CORS_URLS_REGEX = r'^/housing_api/.*$'
```

**Comment by DingoEatingFuzz** [comment]
I made that above change I requested locally and everything was happy dandy for the front-end.

Clearly not the final design, but data from the API on the front-end:

image

File: backend/backend/settings.py (hunk 1 above)
Comment (jaymcgrath) [inline backend/backend/settings.py]: OK, I made these changes and pushed it up to my fork, it should show up in the PR.

**Pull request merged**